rental.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'book'
+
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- [ ] And here in the Rental class we setup the belongs to relation for Rental it belongs to a Book and to a Person, [learn more about it here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) 

The rental is adding itself to the @book achieving the 'belongs-to' association
   ` book.rentals << self`

The rental is adding itself to the @person the 'belongs-to' association
   ` person.rentals << self`